pkg/kubelet/apis/kubeletconfig/v1alpha1/defaults.go

```diff
@@ -50,16 +50,16 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 		obj.ConfigTrialDuration = &metav1.Duration{Duration: 10 * time.Minute}
 	}
 	if obj.Authentication.Anonymous.Enabled == nil {
-		obj.Authentication.Anonymous.Enabled = boolVar(true)
+		obj.Authentication.Anonymous.Enabled = boolVar(false)
 	}
 	if obj.Authentication.Webhook.Enabled == nil {
-		obj.Authentication.Webhook.Enabled = boolVar(false)
+		obj.Authentication.Webhook.Enabled = boolVar(true)
```

Comment:
This will only work if you have a kubeconfig specified, right? Do we need to make this conditional on whether you are in standalone mode?

Reply:
It won't be a silent failure: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/auth.go#L73

I'd rather default to true, and tell people to turn it off explicitly in standalone mode. Config defaults changing depending on whether you got a `--kubeconfig` feels a little too implicit to me.

Reply:
ok